core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -348,40 +348,53 @@ private function setCacheHeaders(Response $response)
 		/** @var PageModel $objPage */
 		global $objPage;
 
+		// Do not cache response if caching was not configured at all and make sure the response is private.
```

Comment:
> was not configured at all

Thats not correct, the value is `0` if it **was** configured to be disabled.

Reply:
It's still correct. If it was not configured it's false. Extended the comment in ea3091400445e9455b6813e3fe7d83e221bacb81.